app/views/provider_interface/account_creation_in_progress.html.erb

```diff
@@ -3,16 +3,17 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-heading-xl">
-      Account creation in progress
+      Your account is not ready yet
     </h1>
     <p class="govuk-body">
-      You are seeing this page because you have successfully signed in to the
-      <%= service_name %> service, but your user account has
-      not yet been associated with your training provider.
+      You're seeing this page because you've signed in
+      to the Apply for teacher training service but your account
```

Comment:
For ‘Apply for teacher training’ we can used the translated string `service_name.apply`

Reply:
Nice one 👍 
Updated in: https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/713/commits/c75ca32902b7e47980e404982b50f0dfda86bcfa